### ISSUE TITLE
Question about the autocoder and training data

### ISSUE THREAD

**Issue by Adele0108** [BioMedAI-UCSC/InverseSR#8]
Hi, there is no details about the encoder and decoder in the paper and code. I would like to ask what the specific model structure of the autocoder? And what's the information of training data, do they include the super resolution ground truth? Looking forward to your reply, thanks~

**Comment by jueqiw** [comment]
Hi, for the model part of this project, I use the model from this [Brain LDM paper](https://arxiv.org/pdf/2209.07162.pdf). You can find the autoencoder architecture [here](https://github.com/BioMedAI-UCSC/InverseSR/blob/main/models/aekl_no_attention.py). I use the images from the IXI dataset, which are high resolution, and we create low-resolution images from them.

Sorry for my late reply.

**Comment by 06Liz** [comment]
from pathlib import Path


# Use environment variables to auto-detect whether we are running an a Compute Canada cluster:
# Thanks to https://github.com/DM-Berger/unet-learn/blob/master/src/train/load.py for this trick.
COMPUTECANADA = False
TMP = os.environ.get("SLURM_TMPDIR")

if TMP:
    COMPUTECANADA = True

if COMPUTECANADA:
    INPUT_FOLDER = Path(str(TMP)).resolve() / "work" / "inputs"
    MASK_FOLDER = Path(str(TMP)).resolve() / "work" / "inputs" / "masks"
    PRETRAINED_MODEL_FOLDER = Path(str(TMP)).resolve() / "work" / "trained_models"
    PRETRAINED_MODEL_DDPM_PATH = (
        Path(str(TMP)).resolve() / "work" / "trained_models" / "ddpm"
    )
    PRETRAINED_MODEL_VAE_PATH = (
        Path(str(TMP)).resolve() / "work" / "trained_models" / "vae"
    )
    PRETRAINED_MODEL_DECODER_PATH = (
        Path(str(TMP)).resolve() / "work" / "trained_models" / "decoder"
    )
    PRETRAINED_MODEL_VGG_PATH = (
        Path(str(TMP)).resolve() / "work" / "trained_models" / "vgg16.pt"
    )
    OUTPUT_FOLDER = Path(str(TMP)).resolve() / "work" / "outputs"
else:
    INPUT_FOLDER = Path(__file__).resolve().parent.parent.parent / "data" / "IXI"
    MASK_FOLDER = Path(__file__).resolve().parent.parent / "masks"
    OASIS_FOLDER = Path(__file__).resolve().parent.parent.parent / "data" / "OASIS"
    PRETRAINED_MODEL_FOLDER = (
        Path(__file__).resolve().parent.parent.parent / "data" / "trained_models"
    )
    这些预训练模型和数据你是怎么处理和下载的 @Adele0108

**Comment by 06Liz** [comment]
Where the decoder is downloaded from
